lib/infra/infra-stack.ts

```diff
@@ -457,6 +468,28 @@ export class InfraStack extends Stack {
           ignoreErrors: false,
         }));
       }
+
+      if (props.enableRemoteStore) {
+        if (props.distributionUrl.includes('artifacts.opensearch.org') && !props.minDistribution) {
+          cfnInitConfig.push(InitCommand.shellCommand('set -ex;cd opensearch; echo "y"|sudo -u ec2-user bin/opensearch-plugin install repository-s3', {
```

Comment:
So can we use `--batch` instead of `echo yes`?

Reply:
Wasn't aware of this, updated. 
Thanks!